### PR TITLE
autotest change tooltip displayed text

### PR DESCRIPTION
**Pull request by kakj-go**
#### What type of this PR

/kind feature

#### What this PR does / why we need it:
autotest change tooltip displayed text

### CODE REVIEW

File: pkg/erda-configs/i18n/autotest.json [hunk 1]
```diff
@@ -6,9 +6,10 @@
     "wb.content.autotest.scene.boolean": "随机一个布尔类型 true",
     "wb.content.autotest.scene.upper": "随机一个大写字符串 ABC",
     "wb.content.autotest.scene.lower": "随机一个小写字符串 abc",
-    "wb.content.autotest.scene.mobile": "随机一个电话号码 10000000000",
-    "wb.content.autotest.scene.digital_letters": "随机一个字符串 abc",
-    "wb.content.autotest.scene.letters": "随机一个字符串 abc",
+    "wb.content.autotest.scene.mobile": "随机一个电话号码 18888888888",
+    "wb.content.autotest.scene.digital_letters": "随机一个字符串 Abc123",
+    "wb.content.autotest.scene.letters": "随机一个字符串 Abc",
+    "wb.content.autotest.scene.character": "随机一个字符 a",
```
Comment (Effet) [inline pkg/erda-configs/i18n/autotest.json:12]: where is this i18n text used?

**Review by Effet — CHANGES_REQUESTED** [review]

File: pkg/erda-configs/i18n/autotest.json (hunk 1 above)
Comment (kakj-go) [reply pkg/erda-configs/i18n/autotest.json:12]: image

autotest scene mock select

**Review by sfwn — APPROVED** [review]

**Pull request merged**